src/constants/FriendMessage.ts

```diff
@@ -1,7 +1,12 @@
 export const FRIEND_REQUEST_SUCCESS = "친구 요청을 성공적으로 보냈습니다.";
 export const SEARCH_SIMILAR_NAMED_USERS_SUCCESS =
   "회원 목록을 성공적으로 얻었습니다.";
-export const NO_SELECTED_USER_FOR_FRIEND_REQUEST =
-  "선택된 회원이 없으므로 친구 요청을 보낼 수 없습니다.";
 export const FRIEND_REQUEST_DUPLICATED_ERROR =
   "이미 친구 요청을 보낸 회원입니다.";
+export const FRIEND_REQUEST_CANCEL_SUCCESS =
+  "친구 요청을 성공적으로 취소했습니다.";
+export const FRIEND_CANCEL_REQUEST_ERROR =
```

Comment:
상수 이름을 바꾸는게 직관적일것 같아요
```suggestion
export const NOT_FOUND_FRIEND_REQUEST_ERROR =
```